docs/_docs/quick-start/getting-started.md

```diff
@@ -25,54 +24,53 @@ apm install nuclide
 > Nuclide is [not currently supported on Windows](/docs/editor/setup/#windows).
 
 While technically optional, in order for features such as [`Quick Open`](#quick-open) to work
-correctly, you will also need to install [`Watchman`](https://facebook.github.io/watchman/) and
-ensure it is on your `$PATH`. There are other
-[recommended installations](/docs/editor/setup/#post-installation) as well.
+correctly, you also need to install [`Watchman`](https://facebook.github.io/watchman/) and
+ensure it is in your `$PATH` enviroment variable. There are other
+[recommended package installations](/docs/editor/setup/#post-installation) as well.
 
 ## Launch
 
-After installation, you launch Nuclide by [opening Atom](/docs/editor/basics/#opening). Once Atom
-is open, you should see the Nuclide home page.
+After installation, launch Nuclide by [opening Atom](/docs/editor/basics/#opening). Once Atom
+is open, you should see the Nuclide Home page.
 
 ![](/static/images/docs/quick-start-getting-started-home-page.png)
 
-The left-hand side pane is the Nuclide file-tree view. The main pane contains introductory
-information about Nuclide and the Quick Launch Menu. This main pane is also where you will be
-editing your files (just like in normal Atom). And there is a bottom status bar to show you error
-and health statistics.
+- The left side-pane is the Nuclide File Tree view.
+- The main pane contains introductory information about Nuclide and the Quick Launch Menu. This is also where you will edit your files (just like in normal Atom).
+- The bottom status bar shows you error and health statistics.
 
 ## Adding a Project
 
-After launching Nuclide, the next common step is to open a project that you would like to work on.
-This could be a Hack, Flow or any other project that has a root directory.
+The first common step after launching Nuclide is to open a project you would like to work on.
+This could be a Hack, Flow, or any other project that has a root directory.
```

Comment:
Probably could link to the actual Hack and Flow pages that we have at nuclide.io. But not a stopper for this pull request, unless you want to when making other updates.